Components/list.css

```diff
@@ -0,0 +1,34 @@
+.list-container {
+  border-radius: 0.3rem;
+  padding: 1rem 32px 1rem 40px;
```

Comment:
Mixing two units is not good practice, and for values greater than 10px prefer to use `rems`

Reply:
I made the changes.